packaging/linux/buildpackage.sh

```diff
@@ -66,7 +66,11 @@ fi
 
 echo "Building core files"
 
-make version VERSION="${TAG}"
+if [ "${PACKAGING_OVERWRITE_MOD_VERSION}" == "True" ]; then
+    make version VERSION="${TAG}"
+else	
+	echo "The mod version will remain unchanged.";
```

Comment:
Ideally this should report the actual mod version "Packaging mod version <x> as <y>" vs "Setting mod version to <y>".  I can help you with a grep command to extract this later in the week if you need help.

Reply:
The mod version is now checked in the makefile.